ruby/vampires.rb

```diff
@@ -22,6 +23,7 @@ def vampire_test
 
 		puts "Our company cafeteria serves garlic bread. Should we order some for you? (y/n)"
 			wants_garlic_bread = gets.chomp
```

Comment:
Something to think on.  As you study methods you will see that we use the passing as parameters.  We should always avoid using `gets.chomp` in a method.